Makefile.d/k8s.mk

```diff
@@ -211,12 +211,17 @@ k8s/metrics/grafana/delete:
 .PHONY: k8s/metrics/jaeger/deploy
 ## deploy jaeger
 k8s/metrics/jaeger/deploy:
-	kubectl apply -f k8s/metrics/jaeger
+	kubectl apply -f https://raw.githubusercontent.com/jaegertracing/helm-charts/jaeger-operator-2.30.0/charts/jaeger-operator/crds/crd.yaml
```

Comment:
can you define operator version as variable?

Reply:
I have fixed :)
https://github.com/vdaas/vald/pull/1682/commits/65e5e53734d5cfb9b6f717cef48fa693b804456f